src/main/docker/kafka/scripts/start-kafka.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
```

Comment:
If this is copied from somewhere please provide a reference at the top and include their copyright header. If you changed this substantially, please include our copyright header and a post-copyright comment regarding the original source.